src/Input.test.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { render, fireEvent, screen } from '@testing-library/react';
+
+import Input from './Input';
+
+describe('Input 컴포넌트에', () => {
+  const inputText = 'hello';
+  const onChange = jest.fn();
+  const onClick = jest.fn();
+
+  beforeEach(() => {
+    render(
+      <Input value="" onChange={onChange} onClick={onClick} />,
+    );
+  });
+
+  test('\'추가\' 버튼이 있다.', () => {
+    screen.getByText(/추가/);
+  });
+
+  test('할 일을 입력 후 추가할 수 있다.', () => {
+    fireEvent.change(screen.getByPlaceholderText(/할 일을 입력해 주세요/), { target: { value: inputText } });
+    fireEvent.click(screen.getByText(/추가/));
+
+    expect(onChange).toBeCalledTimes(1);
+    expect(onClick).toBeCalledTimes(1);
```

Comment:
```suggestion
test('할 일을 입력 후 추가할 수 있다.', () => {
    expect(getByDisplayValue('hello')).not.toBeNull();

    fireEvent.change(screen.getByPlaceholderText(/할 일을 입력해 주세요/), {
      target: {
           value: inputText
    }});

    expect(handleChange).toBeCalledTimes(1);

    fireEvent.click(screen.getByText(/추가/));
    
    expect(handleClick).toBeCalledTimes(1);
```

테스트는 한개로 합칠 수 있을 것 같네요.

어떤 점에서 개선된 것 같나요?